README.adoc

```diff
@@ -119,6 +119,14 @@ can act as starting points or examples:
  provider such as https://stormpath.com/[Stormpath], Active Directory,
  https://developers.google.com/identity/[Google], http://openid.net/[OpenID], etc.
 
+ https://github.com/SMARTRACTECHNOLOGY/devkit-simple-service-example[Create Your Own Microservice]::
```

Comment:
Moving the project to be a module of the DevKit is a viable option I think.  I don't think the information in this tutorial is in the same category as the guides.

There seems a clear line between guides, which are targeted at “Getting Started with the DevKit” vs “Next Steps” which are more advanced topics.

I see this as a more advanced topic, and so belongs in “Next Steps” which are all references to their respective repositories with a short description.
Making this another guide dilutes the message from the Customer Journey (https://smartractechnology.atlassian.net/wiki/display/CORE/DevKit+-+MVP)

This is in the “Extend and relax” section more than the “Check it out” or “Take it for a drive”

The categorization of the documentation should be reflected in its presentation.